packages/web-tracing/package.json

```diff
@@ -53,22 +53,24 @@
   },
   "dependencies": {
     "@grafana/faro-web-sdk": "^1.2.8",
-    "@opentelemetry/api": "^1.4.1",
-    "@opentelemetry/context-zone": "^1.15.2",
-    "@opentelemetry/core": "^1.15.2",
-    "@opentelemetry/exporter-trace-otlp-http": "^0.41.2",
-    "@opentelemetry/instrumentation": "^0.44.0",
-    "@opentelemetry/instrumentation-document-load": "^0.33.0",
-    "@opentelemetry/instrumentation-fetch": "^0.41.2",
-    "@opentelemetry/instrumentation-xml-http-request": "^0.44.0",
-    "@opentelemetry/otlp-transformer": "^0.41.2",
-    "@opentelemetry/resources": "^1.15.2",
-    "@opentelemetry/sdk-trace-base": "^1.15.2",
-    "@opentelemetry/sdk-trace-web": "^1.15.2",
-    "@opentelemetry/semantic-conventions": "^1.15.2"
+    "@opentelemetry/api": "^1.7.0",
+    "@opentelemetry/context-zone": "^1.18.1",
+    "@opentelemetry/core": "^1.18.1",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.45.1",
+    "@opentelemetry/instrumentation": "^0.45.1",
+    "@opentelemetry/instrumentation-document-load": "^0.34.0",
+    "@opentelemetry/instrumentation-fetch": "^0.45.1",
+    "@opentelemetry/instrumentation-xml-http-request": "^0.45.1",
+    "@opentelemetry/otlp-transformer": "^0.45.1",
+    "@opentelemetry/resources": "^1.18.1",
+    "@opentelemetry/sdk-trace-base": "^1.18.1",
+    "@opentelemetry/sdk-trace-web": "^1.18.1",
+    "@opentelemetry/semantic-conventions": "^1.18.1"
   },
   "resolutions": {
-    "@opentelemetry/api-metrics": "^0.29.1"
+    "@opentelemetry/core": "^1.18.1",
+    "@opentelemetry/sdk-trace-base": "^1.18.1",
+    "@opentelemetry/sdk-trace-web": "^1.18.1"
   },
```

Comment:
are these needed? removing doesn't seem to make a difference at least

Reply:
I'll double check thank you Simen!

I appreciate your support a lot. 
This is really great, thank you so much ❤️

Reply:
No they aren't needed.